vehicle/mkv/software/air_control/tests/test_air.py

```diff
@@ -0,0 +1,413 @@
+import pytest
+import math
+import time
+from formula.projects.microhitl import Values as PinValue
+
+# Time constant for motor controller precharge circuit
+MC_TAU = 0.2
+
+
+def reset(iocontroller, pins):
+    iocontroller.write_pin(pins["RESET"][0], PinValue.LOW)
+    time.sleep(0.5)
+    iocontroller.write_pin(pins["RESET"][0], PinValue.HIGH)
+
+
+def initialize_to_idle(canbus, iocontroller, pins):
+    canbus.set_state("pack_voltage", 350)
+    canbus.set_periodic("bms_core", 0.1)
+    canbus.set_state("D1_DC_Bus_Voltage", 0)
+    canbus.set_periodic("M167_Voltage_Info", 0.1)
+
+    iocontroller.write_pin(pins["AIR_P_WELD_DETECT"][0], PinValue.LOW)
+    iocontroller.write_pin(pins["AIR_N_WELD_DETECT"][0], PinValue.LOW)
+    iocontroller.write_pin(pins["SS_TSMS"][0], PinValue.HIGH)
+    iocontroller.write_pin(pins["IMD_SENSE"][0], PinValue.HIGH)
+    iocontroller.write_pin(pins["BMS_SENSE"][0], PinValue.HIGH)
+
+
+"""
+Tests
+"""
+
+
+def test_idle_bms_fault(canbus, iocontroller, pins):
```

Comment:
Docstrings matching the patterns set in other test file would be great

Reply:
Good call.